dataflow/encryption-keys/noxfile_config.py

```diff
@@ -22,7 +22,7 @@
 
 TEST_CONFIG_OVERRIDE = {
     # You can opt out from the test for specific Python versions.
-    "ignored_versions": ["2.7", "3.8", "3.9", "3.10"],
+    "ignored_versions": ["2.7", "3.6", "3.8", "3.9", "3.10"],
```

Comment:
Hi @davidcavazos, is it alright to stop running these tests on 3.6?

Reply:
Yes, that's okay. Thank you!

Reply:
@busunkim96 This looks unresolved; I am going to mark my review as Request Changes because of this, but if I am misunderstanding something let me know.

Reply:
Hi @nicain, this is resolved. The list of python versions are versions that are skipped in tests. I added 3.6 here after checking with David.